doc/source/cluster/index.rst

```diff
@@ -4,42 +4,69 @@
 
 .. _cluster-index:
 
-Ray Cluster Overview
-====================
+Ray Clusters Overview
```

Comment:
Is there an obvious way to lighten things up graphically with an overview image of Ray Clusters features, a GIF of how to start a cluster from command line, or embedding a video about the topic? It'd be great to generate a bit more excitement about the topic somehow.

Reply:
There was a video on how the autoscaler works that @DmitriGekhtman was trying to add back https://github.com/ray-project/ray/issues/24323 but this wouldn't really be intro-level. I can try to produce some asset (image) to lighten up the page. What would you suggest?

Reply:
Sorry, I don't have a good overview of the assets we have.

Reply:
No worries let me think about it for a couple of days and have a separate PR for it, wanna make sure we merge this